docs/experimental-features/security-posture-management.asciidoc

```diff
@@ -0,0 +1,46 @@
+[[security-posture-management]]
+== Cloud security posture management
+
+NOTE: This feature requires Elastic Stack version 8.3.0 or later.
+
+Cloud security posture management (CSPM) and Kubernetes security posture management (KSPM) continuously monitor and compare your cloud and Kubernetes infrastructure against security best practices to help you identify and remediate misconfigurations.
+
+[discrete]
+=== Enable security posture management
+
+The steps to enable this feature differ between Elastic cloud and self-hosted deployments.
+
+[[enable-kspm-on-cloud]]
+[discrete]
+==== Enable KSPM for cloud deployments
+
+First, enable the KSPM flag for your deployment:
+
+* From {kib}, open the main menu and click on **Manage this deployment**.
+
+image::images/kspm-1.png[The Manage deployment button]
+
+1. Under `My deployment`, select **Kibana**.
+2. Under "Instances", click the three-dot menu next to your instance, then click **Edit configuration**.
```

Comment:
@joepeeples correct me if it should be quotes instead of bold -- can't remember what we've done traditionally. 
```suggestion
2. Under **Instances**, click the three-dot menu next to your instance, then click **Edit configuration**.
```

Reply:
going with bold for now